### DISCUSSION TITLE
pnpm monorepo实践 - 愧怍的小站

### DISCUSSION THREAD

**Discussion by giscus** [kuizuo/blog#170] [General]
# pnpm monorepo实践 - 愧怍的小站

使用 pnpm monorepo 实践

https://kuizuo.cn/pnpm-monorepo-practice/#comments

**Comment by JuSanZhi** [comment]
添加 @* 无法安装，报找不到文件

**Reply by kuizuo** [reply]
能否提供相关截图，或者最小复现

**Reply by lizhengxing123** [reply]
如果是mac-zsh，使用 `@\*`就可以了。
